content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cloud service management comes of age with Istio 1.1 performance update
+description: An overview of Istio 1.1 performance.
+publishdate: 2019-03-18
+subtitle: An overview of Istio 1.1 performance improvements
+attribution: Surya V Duggirala (IBM), Mandar Jog (Google), Jose Nativio (IBM)
+keywords: [kubernetes,docker,container,performance,mesh]
+---
+
+Hyper-scale, microservice-based cloud environments have been exciting to build but challenging to manage. Along came Kubernetes (container orchestration) in 2014, followed by Istio (container service management) in 2017. Both open-source projects enable developers to scale container-based applications without spending too much time on administration tasks.
+
+Now, new enhancements in Istio 1.1 deliver scale-up with improved application performance and service management efficiency.
+Simulations using our sample commercial airline reservation application show the following improvements, compared to Istio 1.0.
+
+###### Application performance ######
+
+* up to 30% reduction in application average latency
+* up to 40% faster service startup times in a large mesh
+
+###### Service management efficiency ######
```

Comment:
And change this to:

As well as impressive improvements in service management efficiency: